### ISSUE TITLE
Make _weight_range and _weight_color functions from formatters.html public

### ISSUE THREAD

**Issue by lopuhin** [TeamHG-Memex/eli5#80]
And maybe also some other functions? They are needed if we want to render weights in html similar to how it is done in the html formatter.
Another option would be to use an object instead of (name, weight) tuple, and add ``hsl_color`` attribute to it. I'm not sure which is better, making functions public feels less committing.

**Comment by kmike** [comment]
Another use case: explanation is in JSON, how to show it on a client?

**Comment by lopuhin** [comment]
Right, in this case having colors in JSON looks better. Right now I'm turning this tuple into an object and add colors outside of eli5.

**Comment by kmike** [comment]
It'd be nice to solve it somehow without using HSL colors - users may want to use e.g. a different color scheme.

**Comment by lopuhin** [comment]
I think if the user want to completely change the color scheme, we won't help them much with this functions, but they have the weights (and ``weight_range`` function).
If we keep HSL colors as a list of 3 floats, then they can tweak H and S, right! 👍 
I'd start with just making the functions public and making pairs of (name, weight) an object.
